docs-src/concepts/what-is-the-temporal-platform.md

```diff
@@ -9,12 +9,89 @@ tags:
 ---
 
 The Temporal Platform consists of a [Temporal Cluster](/concepts/what-is-a-temporal-cluster) and [Worker Processes](/concepts/what-is-a-worker-process).
-Together these components create a runtime for Workflow Executions.
+Together these components create a runtime for Workflow Executions; that is-they create a runtime for your application.
 
 ![The Temporal Platform](/diagrams/temporal-platform-simple.svg)
 
 The Temporal Cluster is open source and can be operated by you.
-The Temporal Cloud is a set of Clusters operated by us.
+[Temporal Cloud](/concepts/what-is-temporal-cloud) is a set of Clusters operated by us.
 
-Worker Processes are hosted by you and execute your code.
+Worker Processes are hosted and operated by you and execute your code.
 They communicate with a Temporal Cluster via gRPC.
+
+![Basic component topology of the Temporal Platform](/diagrams/temporal-platform-component-topology.svg)
+
+### Temporal vs traditional
+
+In a traditional system, the service exists to spawn function executions.
+The Temporal Platform exists to facilitate [Workflow Executions](/workflows#workflow-execution).
+
+![Temporal vs Traditional system](/diagrams/temporal-vs-traditional.svg)
+
+Although the two systems seem similar at first glance, they differ in several significant ways.
+
+**Failure**
+
+With a traditional system, a service function execution is both volatile and short-lived.
+
+- If a function execution fails, it's not resumable because all execution state is lost. The longer a function execution awaits, the higher the chance of failure.
```

Comment:
```suggestion
- If a function execution fails, it's not resumable because all execution state is lost.
  The longer a function execution awaits, the higher the chance of failure.
```